lib/services/ngsi/entities-NGSI-v2.js

```diff
@@ -238,6 +238,7 @@ function generateNGSI2OperationHandler(operationName, entityName, typeInformatio
             if (
                 response.statusCode &&
                 response.statusCode === 404 &&
+                errorField.details &&
```

Comment:
Is this code correct taking into account that NGSIv2 responses doesn't include any `details` field?

With regards to `response.statusCode` I understand that `statusCode` is provided by the requests library in the `response` object automatically using the HTTP response code. In that case, that part is fine.

Reply:
If no details is available, what key should be checked in that efforField?

Reply:
done in https://github.com/telefonicaid/iotagent-node-lib/pull/1057/commits/da2873fc1fabd3af2c334819470e6ea1b2146ad0